CatanWizard.html

```diff
@@ -1,12 +1,40 @@
 <!DOCTYPE HTML>
 <html>
 <head>
-    <!-- <link rel="stylesheet" type="text/css" href="cw.css"/> -->
+    <link rel="stylesheet" type="text/css" href="main.css"/>
 </head>
 
 <body>
+    <h1>Generated Board Layout<br></h1>
     <p id="board">Before the script...</p>
 
+    <hr>
+    <form>
+
+        <form id="radios">Number of Players:
+            <input type="radio" name="numPlayers" id="p2" onclick="adjustPlayers();" value="2" checked>2
+            <input type="radio" name="numPlayers" id="p3"onclick="adjustPlayers();" value="3">3
+            <input type="radio" name="numPlayers" id="p4"onclick="adjustPlayers();" value="4">4  
+        </form> 
+        
+        <p id="paragraph1">
```

Comment:
Do you think div tags would be more appropriate here since the p tags are containing more than text?